src/sandstorm/hack-session.capnp

```diff
@@ -104,6 +105,15 @@ interface HackSessionContext @0xe14c1f5321159b8f
   revokeApiToken @5 (tokenId :Text);
   # Revoke (delete) a previously-generated token.
 
+  getUIViewForToken @6 (token :Text) -> (session: Grain.UiSession);
```

Comment:
I think this should return UiView. It should be up to the caller to call newSession() and choose the appropriate WebSession.Params (currently you seem to be filling in invalid dummy values).

Also, nitpick: The 'I' should be lowercase, i.e. `getUiViewForToken`.